src/actions/component.js

```diff
@@ -246,14 +253,100 @@ export const dragComponent = (
 ) => (dispatch, getState) => {
   const state = getState();
   const activesComponents = state.appState.activeComponentsById;
+  const find_Filters = Object.values(activesComponents).filter(
+    element =>
+      element.type === FILTER &&
+      (element.initialMember === idTargetComponent ||
+        element.finalMember === idTargetComponent),
+  );
+  if (find_Filters?.length > 0) {
+    find_Filters.forEach(filter => {
+      if (
+        filter.initialMember === idTargetComponent &&
```

Comment:
I think the statement `filter.initialMember === idTargetComponent` is unnecessary.